tests/test_election_settings.py

```diff
@@ -0,0 +1,79 @@
+import json
+import pytest
+from flask.testing import FlaskClient
+
+from arlo_server import app, db
+from arlo_server.models import Election
+
+from tests.helpers import post_json, put_json
+
+
+@pytest.fixture
+def client() -> FlaskClient:
+    app.config["TESTING"] = True
+    client = app.test_client()
+
+    with app.app_context():
+        db.drop_all()
+        db.create_all()
+
+    yield client
+
+    db.session.commit()
+
+
+@pytest.fixture
+def election_id(client: FlaskClient, request) -> str:
+    rv = post_json(
+        client,
+        "/election/new",
+        {"auditName": f"Test Audit {request.function.__name__}"},
+    )
+    election_id = json.loads(rv.data)["electionId"]
+    yield election_id
+
+
+def test_update_election(client: FlaskClient, election_id: str):
+    rv = put_json(
+        client,
+        f"/election/{election_id}/settings",
+        {
+            "electionName": "An Updated Name",
+            "online": False,
+            "riskLimit": None,
+            "randomSeed": None,
+            "state": None,
+        },
+    )
+    assert rv.status_code == 200, f"unexpected response: {rv.data}"
+    assert json.loads(rv.data) == {"status": "ok"}
+    assert Election.query.filter_by(id=election_id).one().name == "An Updated Name"
```

Comment:
Do you think it's worth trying out updating all the fields?

Reply:
Also the pattern I noticed in other tests was to use the GET API to test the results of the PUT. I kind of like that because it exercises the interface that a consumer uses.

Reply:
Sounds good, will do.